lib/pbench/server/api/resources/__init__.py

```diff
@@ -384,17 +384,22 @@ def convert_list(value: Union[str, List[str]], parameter: "Parameter") -> List[A
     Returns:
         A new list with normalized elements
     """
-    values = value
-    if type(value) is str:
-        values = value.split(",")
-    elif type(value) is not list:
+    values = None
+    if parameter.string_list:
+        if type(value) is str:
+            values = value.split(parameter.string_list)
+        elif type(value) is list:
+            values = [x for e in value for x in e.split(parameter.string_list)]
+    elif type(value) is list:
+        values = value
+    if values is None:
         raise ConversionError(value, f"List of {parameter.name}")
-    etype: "ParamType" = parameter.element_type
+    etype: Union["ParamType", None] = parameter.element_type
     retlist = []
     errlist = []
     for v in values:
         try:
-            retlist.append(etype.convert(v, parameter))
+            retlist.append(etype.convert(v, parameter) if etype else v)
         except SchemaError:
             errlist.append(v)
```

Comment:
It probably won't make a huge difference (since we don't expect the values list to be very long), but consider recasting this code as:
```
    if etype:
        for v in values:
            try:
                retlist.append(etype.convert(v, parameter) if etype else v)
            except SchemaError:
                errlist.append(v)
    else:
        retlist = values
```
That is, if there's no `etype`, then there's no point in going through the loop -- we can just return `values`.